src/loader/BlendLoader.py

```diff
@@ -9,8 +10,43 @@
 
 class BlendLoader(LoaderInterface):
     """
-    Loads entities (everything that can be stored in a .blend file's folders, see Blender's documentation for
-    bpy.types.ID for more info) that match a name pattern from a specified .blend file's section/datablock.
+    This class provides functionality to load entities from a .blend file. A .blend file is a 
+    blender generated  data file that wraps project resources into sections/datablocks. Resources can be
+    loaded individually by name pattern matching or entire datablocks to entire project. For more
+    information about a datablock see Blender's documentation for bpy.types.ID
+    at https://docs.blender.org/api/current/bpy.types.ID.html
+
+    Sections/Datablocks in a .blend File
+
+    +-------------------+
```

Comment:
I think it should be styled as [Available:](https://github.com/DLR-RM/BlenderProc/blob/master/src/object/PhysicsPositioning.py#L54)